### PR TITLE
Add definite form of "temperatur"

### PR DESCRIPTION
**Pull request by ehn**

### CODE REVIEW

**Comment by larsdunemark** [comment]
Looks good, could you maybe add a test sentence for this change so we don't miss it in the future.
Guess it could be like: `var är temperaturen i köket?`

**Comment by ehn** [comment]
> Looks good, could you maybe add a test sentence for this change so we don't miss it in the future. Guess it could be like: `var är temperaturen i köket?`

Sure, is there documentation for how to do this correctly? Should I create a new PR for this?

**Comment by Robin-St** [comment]
This is the relevant file to add tests to:
https://github.com/home-assistant/intents/blob/main/tests/sv/climate_HassClimateGetTemperature.yaml
And this is the documentation for tests:
https://developers.home-assistant.io/docs/voice/intent-recognition/test-syntax/

No need for a new PR, just make the changes to the branch and push them and they will be visible here :)

**Comment by ehn** [comment]
Tests added.

File: tests/sv/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -3,11 +3,13 @@ tests:
   - sentences:
       - "vad är det för temperatur?"
       - "vilken temperatur är det?"
+      - "vad är temperaturen?"
     intent:
       name: HassClimateGetTemperature
   - sentences:
       - "vad är det för temperatur i vardagsrummet?"
       - "vilken temperatur är det i vardagsrummet?"
+      - "vad är temperaturen i köket?"
```
Comment (larsdunemark) [inline tests/sv/climate_HassClimateGetTemperature.yaml:12]: So this doesn't work, the test case is expecting to trigger:
```
   intent:
      name: HassClimateGetTemperature
      slots:
        area: "livingroom"
```
As stated below. But the sentence will trigger `area: "kitchen"` so the testcase fails right now.
So ether change `Köket` to `vardagsrummet` or create a new test case expecting the area of `kitchen`

**Review by larsdunemark — CHANGES_REQUESTED** [review]
Need to pass the testcase

File: tests/sv/climate_HassClimateGetTemperature.yaml (hunk 1 above)
Comment (ehn) [reply tests/sv/climate_HassClimateGetTemperature.yaml:12]: Mea culpa. 🤦‍♂️

**Review by larsdunemark — APPROVED** [review]
Looks good to me

**Pull request merged**